driver_cpl/cime_config/config_component.xml

```diff
@@ -2891,6 +2891,14 @@
     <desc>supplied or computed test id</desc>
   </entry>
 
+  <entry id="TEST_MEMLEAK_TOLERANCE">
+    <type>real</type>
+    <default_value>0.10</default_value>
+    <group>test</group>
+    <file>env_test.xml</file>
+    <desc>Expected memory usage growth for test</desc>
```

Comment:
Should note that this is _relative_ growth - e.g., 0.1 = 10% memory growth. (Right?)